api/src/main/java/org/openmrs/api/ConditionService.java

```diff
@@ -62,6 +62,17 @@
 	@Authorized({ PrivilegeConstants.GET_CONDITIONS })
 	List<Condition> getActiveConditions(Patient patient) throws APIException;
 
+    /**
+	 * Gets a patient's active and inactive conditions
+	 *
+	 * @param patient - the patient to retrieve conditions for
+	 * @throws APIException   
+	 * @return a list of the patient's conditions
+	 * @since 2.4.0-SNAPSHOT
```

Comment:
It should be the version which will be released. Hence 2.4.0

Reply:
@dkayiwa  I updated the PR. :)